### DISCUSSION TITLE
Gallery made, but images are missing

### DISCUSSION THREAD

**Discussion by nightaviation** [thumbsup/thumbsup#335] [Q&A]
Hi!
After running thumbsup (with some help from a lovely online stranger) to create a gallery of my drawings, I keep having this issue where it doesn't seem to process some (most, even, but not all!) images. The links are there, but the images themselves are not on the page (or in the folders itself), nor does clicking on them lead to anything. Do I have something wrong somewhere, or is there a fix? Let me know if I can provide more helpful info, I'm just not sure where to look.  Thanks!

Here's a [link to the gallery](https://nightaviation.neocities.org/art/) just in case.

See attached: 
![image](https://user-images.githubusercontent.com/127921974/230482350-fa0402b4-bf10-45c4-a4be-32ed34587a27.png)

**Comment by rprieto** [comment]
Hi, did it print any warnings when building the gallery? Could you also try re-running thumbsup with `--log debug` and checking for errors?

**Reply by nightaviation** [reply]
I did not get any errors when running it. The only thing that showed up with the debug was `thumbsup:warn dcraw (https://www.cybercom.net/~dcoffin/dcraw/) is not installed. You will not be able to process RAW photos. +0ms` , but most of my images are digitally-made and not actual photographs, so I didn't imagine that would have an effect. Everything else seems to have worked successfully as far as I can tell.
